tabs/createWebauthn.tsx

```diff
@@ -0,0 +1,84 @@
+import { useCallback, useEffect, useState, type DependencyList } from "react"
+
+import { sendToContentScript } from "@plasmohq/messaging"
+
+import {
+  ContentMethod,
+  handleCreateWebauthn,
+  type ContentRequestArguments
+} from "~contents/messages"
+import * as UrlObject from "~tabs/utils/urlObject"
+
+export const CreateWebauthn = () => {
+  const [tabParams, setTabParams] = useState<Record<string, any> | undefined>()
+  const [cred, setCred] = useState<Record<string, any> | undefined>()
+
+  // When a Content Script requests window or tab creation via Service Worker Messaging, passing req.sender.tab.id to the newly created window or tab is recommended. This facilitates sending processed information back to the original Content Script using the specified tabId.
+  useEffect(() => {
+    const urlParams = window.location.href.split("?")
+    const params = UrlObject.objectFromUrlParams(
+      urlParams[urlParams.length - 1].replace(window.location.hash, "")
+    )
+    setTabParams(params)
+  }, [])
+
+  const buttonCreateWebauthnViaContentsDeps: DependencyList = [tabParams]
+  const buttonCreateWebauthnViaContents = useCallback(async () => {
+    const tabId = tabParams.tabId ? tabParams.tabId : undefined
+    const contentReq = {
+      tabId: tabId,
+      name: "",
```

Comment:
Q: Why not use `name` to replace `method` in body?

Reply:
At first, I thought that setting the "name" would define the scope of the message.
Therefore, I try to minimize the use of built-in parameters.
However, recent testing has shown that specifying a "name" does not affect message reception.
I will modify this part of the code.